builtin/game/falling.lua

```diff
@@ -146,8 +146,22 @@ end
 
 local function drop_attached_node(p)
 	local n = core.get_node(p)
+	local drops = core.get_node_drops(n, "")
+	local def = core.registered_items[n.name]
+	if def and def.preserve_metadata then
+		local oldmeta = core.get_meta(p):to_table().fields
+		-- Copy pos and node because the callback can modify them.
+		local pos_copy = {x=p.x, y=p.y, z=p.z}
+		local node_copy = {name=n.name, param1=n.param1, param2=n.param2}
```

Comment:
`n` isn't used later in this function, so a copy would not be strictly required here

Reply:
That is true today, but it might not be true tomorrow.